src/applications/vaos/tests/e2e/community-care.e2e.spec.js

```diff
@@ -0,0 +1,126 @@
+const Timeouts = require('../../../../platform/testing/e2e/timeouts.js');
+const VAOSHelpers = require('./vaos-helpers');
+const Auth = require('../../../../platform/testing/e2e/auth');
+
+const continueButton = '.rjsf [type="submit"]';
+
+module.exports = {
+  after: (client, done) => {
+    client.deleteCookies();
+    client.end();
+    done();
+  },
+  login: client => {
+    const token = Auth.getUserToken();
+
+    VAOSHelpers.initAppointmentListMock(token);
+
+    Auth.logIn(
+      token,
+      client,
+      '/health-care/schedule-view-va-appointments/appointments/',
+      3,
+    ).waitForElementVisible('#appointments-list', Timeouts.slow);
+    // .axeCheck('.main');
+  },
+  'Select new appointment': client => {
+    client
+      .click('#new-appointment')
+      .waitForElementVisible(continueButton, Timeouts.normal)
+      .axeCheck('.main');
+  },
+  'Choose the type of care you need': client => {
+    client.click('[value="323"]').perform(() => {
+      client
+        .click('.rjsf [type="submit"]')
+        .pause(Timeouts.slow)
+        .assert.containsText(
+          'h1',
+          'Choose where you would prefer to receive your care',
+        );
+    });
+  },
+  'Choose where you would prefer to receive your care': client => {
+    client.click('[value="communityCare"]').perform(() => {
+      client
+        .click('.rjsf [type="submit"]')
+        .assert.containsText(
+          'h1',
+          'What date and time would you like to make an appointment?',
+        )
+        .pause(Timeouts.slow);
+    });
+  },
+  'What date and time would you like to make an appointment?': client => {
+    client
+      .click(
+        '.vaos-calendar__calendars button[id^="date-cell"]:not([disabled])',
+      )
+      .perform(() => {
+        client
+          .click('.vaos-calendar__options input[id^="checkbox-0"]')
+          .click('.rjsf [type="submit"]')
+          .assert.containsText(
+            'h1',
+            'Share your community care provider preferences',
+          );
+      });
+  },
+  'Share your community care provider preferences': client => {
+    client.pause(Timeouts.normal);
+    client.click('input[value="983"]').perform(() => {
+      client
+        .click('#root_preferredLanguage [value="english"]')
+        .click('#root_hasCommunityCareProviderYes')
+        .setValue('#root_communityCareProvider_practiceName', 'practice name')
```

Comment:
We have some helpers like `fill` and `selectDropdown` that you can use for filling in text and selecting from dropdowns, they generally have some extra code to make the tests more robust. Our helpers are located in `src/platform/testing/e2e/nightwatch-commands/`